UUID feature in 0.1.10 requires "std" feature to build.

Using v0.1.10 of the crate I get a build error in guid.rs when using only the "uuid" feature.
`win_etw_provider = { version = "0.1.10", features = ["uuid"] }`

Using "uuid" & "std" avoids the break, 
`win_etw_provider = { version = "0.1.10", features = ["std", "uuid"] }`

but "std" was not required in v0.1.9
`win_etw_provider = { version = "=0.1.9", features = ["uuid"] }`

@NateD-MSFT, Quick looks finds #35 as a possible source for this.

**Simple repro**
```
cargo new provider_test
cd provider_test
cargo add win_etw_provider --features=uuid
cargo build

   Compiling win_etw_provider v0.1.10
error[E0599]: no method named `to_owned` found for reference `&[u8; 8]` in the current scope
   --> <snip...>\.cargo\registry\src\index.crates.io-6f17d22bba15001f\win_etw_provider-0.1.10\src\guid.rs:108:29
    |
108 |             data4: fields.3.to_owned(),
    |                             ^^^^^^^^ method not found in `&[u8; 8]`
    |
    = help: items from traits can only be used if the trait is in scope
help: trait `ToOwned` which provides `to_owned` is implemented but not in scope; perhaps you want to import it
    |
1   + use crate::alloc::borrow::ToOwned;
    |
```

--- a/win_etw_provider/src/guid.rs
+++ b/win_etw_provider/src/guid.rs
@@ -1,6 +1,6 @@
-use zerocopy::{AsBytes, FromBytes, FromZeroes};
 #[cfg(feature = "uuid")]
 use crate::alloc::borrow::ToOwned;
+use zerocopy::{AsBytes, FromBytes, FromZeroes};
 
 /// Initializes a `GUID` from literal values.
 #[macro_export]

--- a/win_etw_provider/src/guid.rs
+++ b/win_etw_provider/src/guid.rs
@@ -1,4 +1,6 @@
 use zerocopy::{AsBytes, FromBytes, FromZeroes};
+#[cfg(feature = "uuid")]
+use crate::alloc::borrow::ToOwned;
 
 /// Initializes a `GUID` from literal values.
 #[macro_export]

--- a/win_etw_provider/src/guid.rs
+++ b/win_etw_provider/src/guid.rs
@@ -1,5 +1,3 @@
-#[cfg(feature = "uuid")]
-use crate::alloc::borrow::ToOwned;
 use zerocopy::{AsBytes, FromBytes, FromZeroes};
 
 /// Initializes a `GUID` from literal values.
@@ -107,7 +105,7 @@ impl From<uuid::Uuid> for GUID {
             data1: fields.0,
             data2: fields.1,
             data3: fields.2,
-            data4: fields.3.to_owned(),
+            data4: *fields.3,
         }
     }
 }

